test/devcycle_flutter_client_sdk_test.dart

```diff
@@ -32,6 +32,17 @@ void main() {
 
   group('initialize', () {
     test('initializes with only required params', () async {
+      DVCUser user = DVCUserBuilder().userId('user1').build();
+      DVCClientBuilder().user(user).sdkKey('123').build();
+      expect(methodCall?.method, 'initialize');
+      expect(methodCall?.arguments, {
+        "environmentKey": "123",
+        "user": {"userId": "user1"},
+        "options": null
+      });
+    });
+
+    test('initializes with environmentKey', () async {
```

Comment:
```suggestion
    test('initializes with sdkKey', () async {
```

Reply:
This one specifically tests that the depricated `.environmentKey()` method still works.